src/java/test/org/apache/zookeeper/test/ReconfigExceptionTest.java

```diff
@@ -89,6 +89,7 @@ public void tearDown() throws Exception {
 
     @Test(timeout = 10000)
     public void testReconfigDisabledByDefault() throws InterruptedException {
+        QuorumPeerConfig.setReconfigEnabled(false);
```

Comment:
Given that the test is called "disabled-by-default" Its strange to disable this manually here. I suggest to rename the test to reflect what its doing - checking that when the flag is false reconfig ops aren't allowed.